projects/Aerogramme/default.nix

```diff
@@ -23,13 +23,11 @@
       module =
         { lib, pkgs, ... }:
         {
-          options = {
+          options.services.aerogramme = {
             enable = lib.mkEnableOption "Aerogramme";
             package = lib.mkPackageOption pkgs "aerogramme" { };
           };
           # TODO: add a service definition
-          # NOTE: enabling this fails with "The option `meta.broken' does not exist"
-          # meta.broken = false;
```

Comment:
It's deliberately left as a todo, we need a way to express something doesn't work as intended yet.

Reply:
Made the comment more clear.